### PR TITLE
test: integration test Redis

### PR DESCRIPTION
**Pull request by zucchinidev**
### Checklist:

* ~~[ ] Have you added Draft Release Notes in `docs/draft-release-notes.md`?~~
* [x] Have you followed the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)?

### CODE REVIEW

**Review by pivotal-marcela-campo — CHANGES_REQUESTED** [review]

File: google-redis.yml [hunk 1]
```diff
@@ -70,11 +70,28 @@ provision:
     type: string
     details: The region of the Redis instance.
     default: us-central1
-    constraints:
-      examples:
-      - us-central1
-      - asia-northeast1
-      pattern: ^[A-Za-z][-a-z0-9A-Z]+$
+    enum:
```
[the remark this answers is not in the document]
Comment (zucchinidev) [reply google-redis.yml:73]: Thank you very much, @pivotal-marcela-campo, then I will add these new regions:
* asia-south2 (Delhi)
* asia-southeast2 (Jakarta)
* australia-southeast2 (Melbourne)
* europe-central2 (Warsaw)
* northamerica-northeast2 (Toronto)
* southamerica-west1 (Santiago)
* us-west3 (Salt Lake City)
* us-west4 (Las Vegas)

File: integration-tests/redis_test.go [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+	Describe("updating instance", func() {
+		var instanceID string
+
+		BeforeEach(func() {
+			instanceID, _ = broker.Provision(redisServiceName, customRedisPlan["name"].(string), nil)
+
+			Expect(mockTerraform.FirstTerraformInvocationVars()).To(
+				SatisfyAll(
+					HaveKeyWithValue("instance_id", ContainSubstring("pcf-sb-")),
+					HaveKeyWithValue("instance_id", ContainSubstring("pcf-sb-")),
```
[the remark this answers is not in the document]
Comment (zucchinidev) [reply integration-tests/redis_test.go]: Good catch. Thank you!!!

File: google-redis.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (zucchinidev) [reply google-redis.yml:73]: Done!

**Review by pivotal-marcela-campo — APPROVED** [review]
Looks good! Thanks!

**Pull request merged**